pkg/controller/controller.go

```diff
@@ -104,6 +104,11 @@ const (
 	nodePublishSecretNameKey      = "csiNodePublishSecretName"
 	nodePublishSecretNamespaceKey = "csiNodePublishSecretNamespace"
 
+	// PV and PVC metadata, used for sending to drivers in the  create requests, added as parameters, optional.
+	pvcNameKey      = "csi.storage.k8s.io/created-for/pvc/name"
+	pvcNamespaceKey = "csi.storage.k8s.io/created-for/pvc/namespace"
+	pvNameKey       = "csi.storage.k8s.io/created-for/pv/name"
```

Comment:
Would it make sense to use [the same schema we use in NodePublish](https://github.com/kubernetes/kubernetes/blob/7bbc06fcd436fda2bc82205598e1a91ce568b36f/pkg/volume/csi/csi_mounter.go#L317-L320)?

```
csi.storage.k8s.io/pvc/name
csi.storage.k8s.io/pvc/namespace
csi.storage.k8s.io/pv/name
```

I know CSI drivers that support migration need to translate it to `created-for/` format, still, it would make life of all the non-migrating drivers a bit easier.